sapp/ui/tests/issues_test.py

```diff
@@ -150,8 +150,10 @@ def testWhereCallables(self) -> None:
             builder = Instance(session, latest_run_id)
             issue_ids = {
                 int(issue.issue_instance_id)
-                for issue in builder.where_callables_is_any_of(["%function3"])
```

Comment:
Could you please add tests for the source and sink filtering you added in this PR?

Reply:
We lacked tests for sources and sinks, added them :)